src/passes/Print.cpp

```diff
@@ -835,73 +813,94 @@ struct PrintSExpression : public Visitor<PrintSExpression> {
     printName(curr->value, o) << "))";
   }
   void visitGlobal(Global* curr) {
+    doIndent(o, indent);
     o << '(';
-    printMedium(o, "global ");
-    printName(curr->name, o) << ' ';
-    if (curr->mutable_) {
-      o << "(mut " << printType(curr->type) << ") ";
+    if (curr->imported()) {
+      printMedium(o, "import ");
+      printText(o, curr->module.str) << ' ';
+      printText(o, curr->base.str) << ' ';
+      o << "(global " << curr->name << ' ' << printType(curr->type) << ")";
```

Comment:
Presumably `curr->name` should be printed with `printName` here?

Reply:
Yes, thanks.